test/executor.test.js

```diff
@@ -271,6 +271,62 @@ describe('executor', function() {
         done();
       });
     });
+
+    describe ('for mixins', function() {
+      it('defines `mixins` files', function() {
+        var example = appdir.writeFileSync('mixins/example.js',
+          'module.exports = ' +
+          'function(app, options) { app.mixinExample = true; }');
+
+        var ts = appdir.writeFileSync('mixins/time-stamps.js',
+          'module.exports = ' +
+          'function(app, options) { app.mixinTimeStamp = true; }');
+
+        appdir.writeConfigFileSync('mixins/time-stamps.json', {
+          name: 'Timestamping'
+        });
+
+        boot(app, appdir.PATH);
+        var modelBuilder = app.loopback.modelBuilder;
+
+        var registry = modelBuilder.mixins.mixins;
+        expect(Object.keys(registry)).to.eql(['Example', 'Timestamping']);
+
+        var meta = modelBuilder.mixins.meta;
+        var sourceFiles = meta.map(getSourceFileProperty);
+        expect(sourceFiles).to.eql([example, ts]);
+      });
+
+      it('supports mixin as model', function() {
```

Comment:
I feel the test name does not describe the purpose of test well. How about "applies mixin metadata specified in instructions"?